create/approve.py

```diff
@@ -21,10 +22,10 @@
 
 TEMPLATE = dedent(
     """\
-    user_name: {empty}
+    user_name:
     group_name: {group_name}
     callink_oid: {callink_oid}
-    signatory: {empty}
+    signatory:
```

Comment:
The linter removes trailing spaces in the source, so I put some useless text after the colon in the string and removed it at run time.